Gaijinizer patch/Map217.txt

```diff
@@ -1440,549 +1436,711 @@ I am...!?#GT
 
 どういうわけか、その後に消息を絶っている。
 > CONTEXT: Map217/events/5/pages/2/448/Dialogue/hyakki
-To be precise, it's just the guy who was feeding
-her for months before she was still demonized.
+To be precise, you were the guy who fed her for
+a few months before she turned into a demon.
 
-Somehow, it has been lost since then.#GT
+After that you lost all contact with her.#CheckLater
 > END STRING
```

Comment:
Note

Reply:
This one I'd marked as CheckLater because of the last line where it's not clear who lost contact with who. I assume it was Yami who lost contact with Kazuya, but it sounds a little weird to me for an animal to "lose contact" with a human.

Reply:
It's fine enough. Just add a comma after "that".